framework/src/postprocessors/RelativeSolutionDifferenceNorm.C

```diff
@@ -31,5 +31,5 @@ RelativeSolutionDifferenceNorm::RelativeSolutionDifferenceNorm(const InputParame
 Real
 RelativeSolutionDifferenceNorm::getValue()
 {
-  return _fe_problem.relativeSolutionDifferenceNorm();
+  return dynamic_cast<Transient *>(_app.getExecutioner())->relativeSolutionDifferenceNorm();
```

Comment:
This line doesn't make sense. You are dynamic casting which means that you expect that it might fail, yet you are returning something anyway. Either this is a static cast or you need an else statement.